meta-lxatac-software/recipes-core/bundles/files/hook.sh

```diff
@@ -2,6 +2,39 @@
 
 set -exu -o pipefail
 
+CERT_AVAILABLE_DIR="${RAUC_SLOT_MOUNT_POINT}/etc/rauc/certificates-available"
+CERT_ENABLED_DIR="${RAUC_SLOT_MOUNT_POINT}/etc/rauc/certificates-enabled"
+
+function enable_certificates () {
+	# Ignore the enabled certifcates from the bundle
+	rm "${CERT_ENABLED_DIR}"/*
+
+	for cert in "${CERT_AVAILABLE_DIR}"/*; do
+		cert_name=$(basename "${cert}")
+
+		cert_hash=$(openssl x509 -pubkey -noout -in "${cert}" \
+			| openssl pkey -pubin -outform der \
+			| openssl dgst -sha256 -c -hex \
+			| awk '{print toupper($2)}')
+
+		# Enable certificates that match the hash of the public key(s)
+		# that the current bundle is signed with.
+		# This means that a bundle signed with e.g. an official stable
+		# channel certificate will only be able to install other
+		# bundles from the same release channel.
+		for slot_hash in ${RAUC_BUNDLE_SPKI_HASHES}; do
+			if [ "${slot_hash}" == "${cert_hash}" ]; then
```

Comment:
It's not a per *slot* hash, but for the bundle signature. Perhaps?
```suggestion
		for bundle_hash in ${RAUC_BUNDLE_SPKI_HASHES}; do
			if [ "${bundle_hash}" == "${cert_hash}" ]; then
```